### PR TITLE
feat(solidjs): Create gists view

### PR DESCRIPTION
**Pull request by SDaian**
## Acceptance Criteria

- [X] Should display Gists views 

## Preview 

image

### CODE REVIEW

File: solidjs-tailwind/src/pages/Home.jsx [hunk 1]
```diff
@@ -1,15 +1,33 @@
...
 
+  const [data] = createResource(() => {
+    try {
+      return useOctokit()
+        .rest.users.getAuthenticated()
+        .then((response) => response.data);
+    } catch {
+      return Promise.resolve({});
+    }
+  });
```
Comment (danecando) [inline solidjs-tailwind/src/pages/Home.jsx:18]: @AllStackDev1 mentioned in the last PR but it looks like we're using the gql api with the fetch util for all the data fetching in this starter kit. We might want to get rid of Ocotokit if thats the case

**Review by danecando — COMMENTED** [review]
Looks good to me. I left one comment about the Octokit thing. I'm guessing the data fetching will be added to Container in another PR?

File: solidjs-tailwind/src/pages/Home.jsx (hunk 1 above)
Comment (SDaian) [reply solidjs-tailwind/src/pages/Home.jsx]: yep @danecando that's right, we should get rid of the Octokit, also the PR was rebased within `main` to have the latest commits.

**Review by danecando — APPROVED** [review]

**Review by AllStackDev1 — APPROVED** [review]
LGTM

**Pull request merged**